pom.xml

```diff
@@ -68,7 +68,8 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.compiler.release>8</maven.compiler.release>
 
-        <opensearch.version>1.1.0</opensearch.version>
+        <opensearch.version>1.1.0-SNAPSHOT</opensearch.version>
+        <version>1.1.0.0</version>
```

Comment:
This is referenced on the line 37 that is security plugin version. Same as [this commend](https://github.com/opensearch-project/security/pull/1409/files#r695406712), doesn't seem to need a change at here.

Reply:
Are you saying I need to make two separate PRs, one for the version bump and one for building snapshots by default?

Reply:
It had to move to properties or you cannot set it via a profile.